lib/sles4sap_publiccloud_basetest.pm

```diff
@@ -0,0 +1,56 @@
+# SUSE's openQA tests
+#
+# Copyright SUSE LLC
+# SPDX-License-Identifier: FSFAP
+# Maintainer: QE-SAP <qe-sap@suse.de>
+# Summary: Base test providing cleanup, post fail and post run hooks for qe-sap-deployment project.
```

Comment:
for tests using qe-sap-deployment project.

Reply:
Changed